server/public/js/ops.js

```diff
@@ -0,0 +1,4 @@
+'use strict'
+
+const chapters = document.querySelector('.chapters');
```

Comment:
I see that this const isn't exported - is it being used anywhere?

Reply:
Removed